druid/examples/flex.rs

```diff
@@ -13,17 +13,39 @@
 // limitations under the License.
 
 //! Demonstrates alignment of children in the flex container.
+//! This is more of a widget showcase where you can turn all the knobs
+//! and see how it behaves and less of an example.
```

Comment:
```suggestion
//! This example showcases the full set of functionality of flex, giving you 
//! knobs to change all the parameters. 99% of the time you will want to
//! hard-code these parameters, which will simplify your code considerably.
```

What do you think? Is this what you're getting at with your comment? Feel free to push back if you prefer the original.